### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Hugoyhu**
There are loopholes in the current- as "excessive", "non-beneficial" and "commercials".

The current stance of slack appears to be against any advertisement not permitted by HQ, my updated line is likely more sufficient.

### CODE REVIEW

**Comment by sampoder** [comment]
cc: @hackclub/community-team & @hackclub/core

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ These actions are explicitly forbidden in Hack Club spaces:
   - discrimination based on age, nationality, race, (dis)ability, gender (identity or expression), sexuality, religion, or similar personal characteristic;
   - bullying or systematic harassment;
   - unwelcome sexual advances, including sexually explicit content.
-- Excessive advertisement for unnecessary or non-beneficial commercial products and services.
+- Advertisement for events, companies, organizations, etc- unless specifically given permission by Staff, Community Team, or a Hack Club Community Game Designer.
```
Comment (sampoder) [inline CODE_OF_CONDUCT.md:68]:
```suggestion
- Advertisement for events, companies, organizations, etc- unless specifically given permission by Hack Club HQ.
```

**Comment by cjdenio** [comment]
I think this looks good!

**Comment by juliegoat** [comment]
It looks good to me too - makes sense! @zfogg

**Comment by zfogg** [comment]
i am reviewing this thank you @Hugoyhu may corgis surround you for all of your days

**Review by zfogg — CHANGES_REQUESTED** [review]

**Review by zfogg — APPROVED** [review]
lgtm

**Pull request merged**